lib/validators/liveValidator.ts

```diff
@@ -656,7 +691,11 @@ export class LiveValidator {
         onlyFiles: true,
         unique: true
       })
-      log.debug(
+      log.info(
```

Comment:
Could you merge log call and loggingCallback call into one private function?

Reply:
Resolved.